public/viz-list.csv

```diff
@@ -0,0 +1,28 @@
+﻿id,released,viz-type,group,title,description,img,alt,url,tags,code

```

Comment:
Not an urgent thing: but let's do some thinking about where the main list for this lives.  A csv in Sharepoint that we update and put in as csv that gets updated periodically, or whether it's a json file inside this repo, or whatever will be most usable/least likely to get destroyed or antiquated.

Reply:
I think a shared spreadsheet that we can edit easily will probably make the most sense/be the easiest to maintain. Then we can export to json/csv/however we want to use it in vue. These columns I've added definitely need more careful thought